start_emu_headless.sh

```diff
@@ -57,68 +57,80 @@ function hidden_policy() {
   adb shell "settings put global hidden_api_policy_pre_p_apps 1;settings put global hidden_api_policy_p_apps 1;settings put global hidden_api_policy 1"
 };
 
-function access_emulator_with_adb() {
-  if test -s /tmp/failed; then
-    echo "Skip"
-  else
-    "adb" shell ls || true
-  fi
-};
-
 function check_emulator_focus() {
```

Comment:
The "check_emulator_focus" function, as far as I see, duplicates the behavior of the previous "launch_emulator" function. It will always succeed because it is only executed after "launch_emulator" which will only return a value of 1 if the emulator has fully booted without any issues.
correct me if I'm mistaken